model/qti/OutcomeDeclaration.php

```diff
@@ -22,7 +22,6 @@
 
 namespace oat\taoQtiItem\model\qti;
 
-use oat\taoQtiItem\model\qti\OutcomeDeclaration;
 use oat\taoQtiItem\model\qti\VariableDeclaration;
 use oat\taoQtiItem\model\qti\Item;
 use oat\taoQtiItem\model\qti\response\Composite;
```

Comment:
All of those use statements are unused. Maybe they could be removed as well (the same thing was done to ` model/qti/attribute/Interpretation.php`)?